ims/database/image.py

```diff
@@ -46,12 +46,10 @@ def delete_with_name_from_project(self, name, project_name):
     def fetch_id_with_name_from_project(self, name, project_name):
         try:
             self.connection = DatabaseConnection()
-            images = self.connection.session.query(Image). \
-                filter(Image.project.has(name=project_name)).filter_by(name=name)
-            if images.count() == 0:
-                return None
-            else:
-                return images.first().id
+            image = self.connection.session.query(Image). \
+                filter(Image.project.has(name=project_name)).filter_by(name=name).one_or_none()
+            if image is not None:
+                return image.id
```

Comment:
There is no need to check if the image is None. Just return it. You are returning nothing anyway in that case.